client/task-list/tasks/payments/paypal.js

```diff
@@ -2,20 +2,16 @@
  * External dependencies
```

Comment:
Given it's a very basic process now that only installs the plugin, I deleted a lot of unused code in here.